File: 백준/Bronze/1259. 팰린드롬수/팰린드롬수.js
let input = require("fs").readFileSync(0).toString().trim().split("\n");
inputArr = input.map(a => a.split(""));
let result = '';
for (i of inputArr) {
    if (String(i.slice().reverse()) === String(i)) {
        if (i[0] === '0')
            continue;
        result += 'yes\n';
    } else {
        result += 'no\n';
    }
}
console.log(result);

File: 백준/Bronze/1259. 팰린드롬수/팰린드롬수.java
import java.io.*;

public class Main {
    public static void main(String[] args) throws Exception {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        StringBuilder result = new StringBuilder();
        while (true) {
            StringBuilder sb = new StringBuilder(br.readLine());
            if (sb.toString().equals("0"))
                break;
            if (sb.toString().equals(sb.reverse().toString())) {
                result.append("yes").append("\n");
            } else {
                result.append("no").append("\n");
            }
        }
        System.out.println(result.toString());
    }
}
